### PR TITLE
newusers: add page

### PR DESCRIPTION
**Pull request by talented**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/newusers.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# newusers
+
+> Update and create new users in batch mode.
+> More information: <https://manned.org/useradd>.
```
Comment (bl-ue) [inline pages/linux/newusers.md:4]: Did you mean to specify the link for `useradd`?
```suggestion
> More information: <https://manned.org/newusers>.
```

File: pages/linux/newusers.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- It reads from a file with the following format:
+
+`<username>:<passwd>:<uid>:<gid>:<user_info>:<home_dir>:<default_shell>`
```
Comment (bl-ue) [inline pages/linux/newusers.md:8]: Hmm.  This format reference is totally useful, but it's not a command.  In tldr pages we only give some demonstrations on how to run a command, with the various arguments it accepts.

File: pages/linux/newusers.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create users.txt file and add account details (Multiple entries allowed):
+
+`ozzy:Linmaster1:1003:1003:Admin User:/home/ozzy:bin/bash`
```
Comment (bl-ue) [inline pages/linux/newusers.md:12]: Like I said above, this example, while very useful, is not a command and had best be added to a cheat repository, such as the projects listed [here](https://github.com/tldr-pages/tldr#similar-projects).

File: pages/linux/newusers.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Set required permissions:
+
+`chmod 600 users.txt`
```
Comment (bl-ue) [inline pages/linux/newusers.md:16]: This command example doesn't demonstrate how to use the `newusers` command and should be removed.

File: pages/linux/newusers.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run with input file:
+
+`{{newusers}} users.txt`
```
Comment (bl-ue) [inline pages/linux/newusers.md:20]: You've got the token syntax mixed up—it goes around the variable arguments, not the command itself.
```suggestion
`newusers {{users.txt}}`
```

**Review by bl-ue — COMMENTED** [review]
Thank you for the new page, @talented!  As explained in my below comments, while very helpful, the examples in this page are not focused on ways to run the command.  In the tldr-pages project we try to show useful examples on how to run the command itself, along with its flags, arguments, etc.a

**Comment by marchersimon** [comment]
@talented are you still working on this?

File: pages/linux/newusers.md (hunk 2 above)
Comment (talented) [reply pages/linux/newusers.md:8]: I understand your concern but without giving this information, it doesn't make sense to add it to tldr.

**Comment by talented** [comment]
Without giving some detailed information on how to use this command other than just running it with parameters, it really doesn't make sense to add it to tldr. We should better cancel this PR. Thanks for reviewing.

**Comment by bl-ue** [comment]
I'm afraid we'll have to...it's really a very nice page, but tldr-pages is about commands.

**Comment by bl-ue** [comment]
If you happen to be interested in contributing @talented, you can search our [Let's document issues](https://github.com/tldr-pages/tldr/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22), or contribute to [page translations](https://lukwebsforge.github.io/tldri18n/) 😉

**Pull request closed without merging**